### ISSUE TITLE
With Denon set pimatic fails to start 

### ISSUE THREAD

**Issue by leader21** [saxn-paule/pimatic-pio-remote#1]
having this issue when starting up 
![](http://www.youscreen.de/nziapoqz61.jpg)

my plugin : 

```
    {
      "plugin": "pio-remote",
      "host": "192.168.178.67",
      "port": 23,
      "brand": "denon"
    },
```

my device 

```
    {
      "class": "AVRSensor",
      "id": "avrsensor",
      "name": "AVR",
      "attributes": [
        {
          "name": "display"
        },
        {
          "name": "vol"
        }
      ],
      "xAttributeOptions": [
        {
          "name": "vol",
          "displaySparkline": false,
          "hidden": false
        }
      ]
   },   
```

any idea what this could cause and how to solve?

**Comment by saxn-paule** [comment]
Yes, I'm sorry. There was a typo. I fixed it in the develop branch. Please try again.

**Comment by saxn-paule** [comment]
published new version 0.4.0 please retest.
